cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/validation/admins/AdminsValidator.kt

```diff
@@ -17,17 +17,12 @@ object AdminsValidator : Validator {
             }
 
     private fun validateAdmins(config: Admins): Boolean {
```

Comment:
As it turned out after writing JUnit - we had a bur in this method - conclusion:

**Write JUnit tests** - especially when method have some `if-else`